src/xdotoolify.js

```diff
@@ -330,12 +330,32 @@ _Xdotoolify.prototype._mouseup = function(mouseButton, checkAfter = false) {
 _Xdotoolify.prototype.mouseup = function(mouseButton) {
   return this._mouseup(mouseButton, true);
 };
-_Xdotoolify.prototype._wheeldown = function(checkAfter = false) {
+_Xdotoolify.prototype._jitter = function() {
+  this._mousemove(
+    {relx: 0, rely: 0}
```

Comment:
e.g. if I am just reading this code and not looking into mousemove, this looks obviously incorrect, but is actually correct. Making the jitter argument to mousemove explicit would align the expectation with reality.

Reply:
Changed to call `_jitter`.